src/types/index.js

```diff
@@ -1,16 +0,0 @@
-const path = require('path');
-const { print } = require('graphql');
-const fs = require('fs');
-const { loadFilesSync } = require('@graphql-tools/load-files');
-const { mergeTypeDefs } = require('@graphql-tools/merge');
-
-const loadedFiles = loadFilesSync(__dirname, { extensions: ['graphql'], recursive: true });
-
-const typeDefs = mergeTypeDefs(loadedFiles);
-
-const printedTypeDefs = print(typeDefs);
-fs.writeFileSync('schema.graphql', printedTypeDefs);
-
-module.exports = {
-  typeDefs
-}
```

Comment:
also why can't we keep the logic for types in here?

Reply:
It throws some error when I tried to do it in this file.

Reply:
what was the error?

Reply:
```
/Users/danial.javanmardi/src/elasticpath-graphql-server/dist/schema/src/addResolversToSchema.js:18
    Object.keys(resolvers).forEach(typeName => {
                           ^
Error: "Product" defined in resolvers, but not in schema
    at /Users/danial.javanmardi/src/elasticpath-graphql-server/dist/schema/src/addResolversToSchema.js:35:23
    at Array.forEach (<anonymous>)
    at addResolversToSchema (/Users/danial.javanmardi/src/elasticpath-graphql-server/dist/schema/src/addResolversToSchema.js:18:28)
    at schemaTransforms (/Users/danial.javanmardi/src/elasticpath-graphql-server/dist/schema/src/makeExecutableSchema.js:66:41)
    at /Users/danial.javanmardi/src/elasticpath-graphql-server/dist/schema/src/makeExecutableSchema.js:108:65
    at Array.reduce (<anonymous>)
    at Object.makeExecutableSchema (/Users/danial.javanmardi/src/elasticpath-graphql-server/dist/schema/src/makeExecutableSchema.js:108:29)
    at Object.<anonymous> (/Users/danial.javanmardi/src/elasticpath-graphql-server/src/index.ts:32:16)
    at Module._compile (node:internal/modules/cjs/loader:1109:14)
    at Module.m._compile (/Users/danial.javanmardi/src/elasticpath-graphql-server/node_modules/ts-node/src/index.ts:1056:23)
[nodemon] app crashed - waiting for file changes before starting...
```

Reply:
I've fixed it, it was from the path. 

I did not want to spend time on it initially, and imho I prefer to have only `graphql` in this folder.

Reply:
I had to revert [this commit](https://github.com/elasticpath/elasticpath-graphql-server/pull/87/commits/2215ecf41748c591c12832b338350242eaafef77) as it failed in CI. Let me know how you feel about it, I can fix CI tomorrow if you have a strong opinion about it.